pysmartthings/device.py

```diff
@@ -228,6 +227,13 @@ def values(self) -> Dict[str, Any]:
             lambda: None, {k: v.value for k, v in self._attributes.items()}
         )
 
+    @property
+    def disabled_components(self) -> []:

```

Comment:
How would you suggest to deal with disabled capabilities then? The smartthings platform seems to respect the disabled platforms capability in the app and website. For most users it will be quite confusing to have capabilities present but not working, should these then always be manually disabled? 

Some of these custom capabilities seem to be to some extent essential functionality to make full use of the smartthings platform in some way a door should be opened to them.

Reply:
That's my question too; would you accept this moved to ha/core, assuming we check the data type first?